test/verify/check-system-services

```diff
@@ -285,8 +301,19 @@ Unit=test.service
         b.wait_in_text(self.svc_sel('test-onboot.timer') + ' .service-unit-last-trigger', "unknown")  # last trigger
         self.run_systemctl(user, "stop test-onboot.timer")
 
-        # Selects Paths tab - the test VM does not have any user path units
         if not user:
+            self.goto_service("deleteme.timer")
+            self.do_action("Delete")
+            self.wait_page_load()
+            self.assertNotIn("deleteme.timer", m.execute("systemctl list-timers"))
```

Comment:
Please drop the `.timer` -- it *could* be that `list-timers` just omits the suffix. Also check that both files are gone from disk.